research/train_model.py

```diff
@@ -0,0 +1,11 @@
+import multiprocessing
+
+if __name__=="__main__":
+    model_name = "model.mdl"
+    model_input = "wiki.he.text"
+
+    from gensim.models import Word2Vec
+    from gensim.models.word2vec import LineSentence
+    model = Word2Vec(LineSentence(model_input), sg=1, vector_size=100, window=5, min_count=20, workers=multiprocessing.cpu_count())
+    model.init_sims(replace=True)
```

Comment:
I think this deprecated

Reply:
its kind of strange... it seems deprecated yet it does normalize all distances... whatever you think